wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -81,6 +81,17 @@ struct ConversationEventPayloadProcessor {
             Logging.eventProcessing.error("Conversation creation missing timestamp in event, aborting...")
             return
         }
+        guard let conversationID = payload.id ?? payload.qualifiedID?.uuid else {
+            Flow.createGroup.fail(ConversationEventPayloadProcessorError.noBackendConversationId)
+            Logging.eventProcessing.error("Conversation creation missing conversationID in event, aborting...")
+            return
+        }
```

Comment:
Does it make sense to add a unit test for this scenario? In case of refactorings or maintenance it might help in future :)

Reply:
I've added the missing unit tests

Reply:
Thanks!